config/gail_config.yaml

```diff
@@ -32,7 +32,7 @@ Pyramids:
     max_steps: 5.0e5
     num_epoch: 3
     behavioral_cloning:
-        demo_path: UnitySDK/Assets/ML-Agents/Examples/Pyramids/Demos/ExpertPyramid.demo
+        demo_path: Project/Assets/ML-Agents/Examples/Pyramids/Demos/ExpertPyramid.demo
```

Comment:
All demo files are now in the project.  Should we put them in the `Samples` section of the package?